autonomy/deploy/generators/docker_compose/base.py

```diff
@@ -286,13 +281,14 @@ def generate(
         network = Network(name=network_name)
         image_version = image_version or self.service_builder.service.agent.hash
         if self.dev_mode:
-            image_version = "dev"
+            runtime_image = DEVELOPMENT_IMAGE
+        else:
+            runtime_image = OAR_IMAGE.format(
+                image_author=self.image_author,
+                agent=self.service_builder.service.agent.name,
+                version=image_version,
+            )
```

Comment:
Using a separate image for development mode, this will be faster compare to the previous one